backports/gflags.v3.8/APKBUILD

```
@@ -0,0 +1,28 @@
+# Contributor: Florent Ferreri <florent@seqsense.com>
+pkgname=gflags
+pkgver=2.2.2
+pkgrel=0
+pkgdesc="The gflags package contains a C++ library that implements commandline flags processing."
+url="https://github.com/gflags/gflags"
+arch="noarch"
+license="BSD-3-Clause"
+options="!check"
+makedepends="cmake"
+subpackages="$pkgname-dev"
```

```suggestion
subpackages="$pkgname-dev $pkgname-doc"
```
for license file installation.